docs/antora-playbook.yml

```diff
@@ -0,0 +1,52 @@
+antora:
+  extensions:
+    - '@springio/antora-extensions/partial-build-extension'
+    - require: '@springio/antora-extensions/latest-version-extension'
+    - require: '@springio/antora-extensions/inject-collector-cache-config-extension'
+    - '@antora/collector-extension'
+    - '@antora/atlas-extension'
+    - id: pdf-extension # pdf-extension is enabled from antora-docs.gradle
+      require: '@antora/pdf-extension'
+      enabled: false
+    - require: '@springio/antora-extensions/root-component-extension'
+      root_component_name: 'reactor-netty'
+site:
+  title: Reactor Netty Reference Guide
+  url: https://pderop.github.io/test-antora-pages/docs/netty/reference/
```

Comment:
Is this URL correct?

Reply:
no, I have corrected it in https://github.com/reactor/reactor-netty/pull/3120/commits/3e12ac625eb3fac7c849a80f04b5c22ae594a362